[feat] Add pipeline example

jwkim1993 opened the pull request:
- Add task and taskrun examples which are used for pipeline

jwkim-tmax committed:
[feat] Add pipeline example

- Add task and taskrun examples which are used for pipeline

cqbqdd11519 commented on example/pipeline/3rd-task.yaml:
```diff
@@ -0,0 +1,114 @@
+  namespace: default
+spec:
+  params:
+  - name: registryAddr
```
요 아이는 어디에 쓰이는 아이인가요?

jwkim1993 replied:
레지스트리를 동적으로 받아서, 사용하는 이미지의 레지스트리를 동적으로 바꾸려고 했는데, 하다보니 어차피 실제 파이프라인 구성할 때도 박혀있을거라고 생각해서 안썼네요. 삭제하겠습니다.

cqbqdd11519 commented on example/pipeline/3rd-task.yaml:
```diff
@@ -0,0 +1,114 @@
+      value: Approval is requested for CI/CD
+    - name: MAIL_CONTENT
+      value: |
+        Suceed to build.
```
Suceed 오타ㅎ..

cqbqdd11519 commented on example/pipeline/approval-sa.yaml:
```diff
@@ -0,0 +1,36 @@
+  name: approval-sa
+  namespace: default
+secrets:
+  - name: ck-registry
```
ck-registry 접근 권한은 왜 필요한가요?

jwkim1993 replied:
당연히 pull image가 되어야하는데 pull이 안되어서 sa에 docker hub 접근할 권한을 준거였는데, 그냥 단순하게 태그가 없었던것 같습니다. 삭제하겠습니다.

jwkim-tmax committed:
[fix] Fix typo and remove unused parameter

- Fix typo "Suceed"
- Remove unused parameter "registryAddr"
- Remove unused secret "ck-registry"
